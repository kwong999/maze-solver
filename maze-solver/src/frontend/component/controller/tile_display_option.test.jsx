import React from 'react';
import TileDisplayOption from './tile_display_option';
import renderer from 'react-test-renderer';

test('TileDisplayOption with checkbox to activate "Searched Tiles"', () => {
  const component = renderer.create(
    <TileDisplayOption
      displaySearchedTile={() => { }}
      changeDisplaySearchedTile={() => { }}
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})