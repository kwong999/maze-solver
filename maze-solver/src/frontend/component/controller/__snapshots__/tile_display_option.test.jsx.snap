// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TileDisplayOption with checkbox to activate "Searched Tiles" 1`] = `
<div
  className="tile-display-option maze-action"
>
  <div>
    <p>
      Display Searched Tile: 
    </p>
    <input
      checked={[Function]}
      className="checkbox"
      onChange={[Function]}
      type="checkbox"
    />
  </div>
</div>
`;
